main.nf

```diff
@@ -111,6 +115,9 @@ def helpMessage() {
       --bamutils_clip_left / --bamutils_clip_right  Specify the number of bases to clip off reads
       --bamutils_softclip           Use softclip instead of hard masking
 
+    Genotyping
+      --genotyping_haplo_caller     Genotype calling using GATK HaplotypeCaller
```

Comment:
I'd say just do Sex Determination + X Chromosomal contamination tests, lets do the Genotyping in a separate pull request once this is done to keep things cleaner 👍